.gitignore

```
@@ -10,6 +10,7 @@ ctime_tests
 ecdh_example
 ecdsa_example
 schnorr_example
+ellswift_example
```

nit: I understand that this doesn’t adhere to the surrounding code, but, strictly speaking, it should be
```suggestion
/ellswift_example
```

---

Yeah, feel free to adjust this for the binaries, but I guess either variant is fine (we also have the pattern `*.exe`, which may in rare cases a bit too coarse`...)